libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-details.component.ts

```diff
@@ -90,10 +93,7 @@ import { ElectricityMarketMeteringPointType } from '@energinet-datahub/dh/shared
                 {{ address?.cityName | dhEmDashFallback }}
               </div>
 
-              <dh-actual-address
-                [isActualAddress]="!!address?.darReference"
-                class="watt-space-stack-m"
-              />
+              <dh-actual-address [isActualAddress]="actualAddress()" class="watt-space-stack-m" />
```

Comment:
suggestion: just checked the `dh-actual-address` component and it looks like instead of creating a `actualAddress` `computed` in those places where the component is used, it will be better to modify `dh-actual-address` to take the `washInstructions` as an Input (replacing the `isActualAddress`). Then we can only check `washInstructions ===  WashInstructions.Washable` once.

Reply:
updated